src/components/checkbox/checkbox.scss

```diff
@@ -0,0 +1,89 @@
+$__checkbox-color: #004890;
+.__checkbox-label {
+  display: inline-block;
+  position: relative;
+  cursor: pointer;
+}
+
+.__checkbox-label input {
+  opacity: 0;
+  cursor: pointer;
+  display: none;
+}
+
+.__checkbox-label .__checkbox-custom {
+  position: relative;
+  display: inline-block;
+  margin-right: 10px;
+  width: 18px;
+  height: 18px;
+  background-color: transparent;
+  bottom: -3px;
+  transition: all 0.3s ease-out;
+  -webkit-transition: all 0.3s ease-out;
+  -moz-transition: all 0.3s ease-out;
+  -ms-transition: all 0.3s ease-out;
+  -o-transition: all 0.3s ease-out;
```

Comment:
Shouldn't we use **autoprefixer** on build step instead of declaring manually every prefix?

Reply:
removed